Tests/Clappr_Tests/AVFoundationPlaybackTests.swift

```diff
@@ -1274,6 +1274,20 @@ class AVFoundationPlaybackTests: QuickSpec {
                         expect(nowPlayingService.countOfCallsOfSetItems).toEventually(equal(0))
                     }
                 }
+
+                context("when avplayer has playerItem and is ready to play") {
+
+                    it("calls loadMetadata when the player is ready to play") {
```

Comment:
Given the context, I think we can get rid of the redundancy here:
`it("calls loadMetadata when the player is ready to play")`
could be just
`it("calls loadMetadata")`

Reply:
done!